util/mirror-tarballs

```diff
@@ -850,6 +850,17 @@ cd ..
 
 #################################
 
+ver=0.01rc1
```

Comment:
I just pushed the v0.01 tag for lua-table. Please use the formal version and merge this PR. Thanks!